mypy/test/data.py

```diff
@@ -65,6 +65,8 @@ def parse_test_cases(
             tcout = []  # type: List[str]
             if i < len(p) and p[i].id == 'out':
                 tcout = p[i].data
+                if p[i].arg == 'pathfix':
+                    tcout = [s.replace('/', os.path.sep) for s in tcout]
```

Comment:
The only way to be sure is to survey all [out] sections of all test files. Or at least sample a bunch of different ones (some have a different format, the test prints the repr() of some Node).

I just worry that your current approach will mean that every time a new test requires [out pathfix] it will first be committed without the pathfix and then after some time someone will report that it's broken on Windows and someone has to remember how to fix it.